src/styles/utils/variables.scss

```diff
@@ -0,0 +1,15 @@
+/* colors */
+$main-accent: #060b35;
+$secondary-color: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+$gray-elements: #e5e5e5;
+$yellow-stars-color: #ffde6a;
+
+/* catalog grip */
+$card-width: 200px;
+$gapY: 48px;
+$gapX: 50px;
```

Comment:
are you sure about the gap values?
46px and 48px seem the right ones

Reply:
another impact is that tests won't pass - gap is not supported by testing browser
so you have to work it around with margins (say, top and right and prevent first 4 cards from having margin-top, while other 4 will have both)